packages/eslint-config/package.json

```diff
@@ -25,21 +25,21 @@
     "prepack": "node ../../tools/clean-package-json.mjs"
   },
   "dependencies": {
-    "@eslint-react/eslint-plugin": "^1.15.2",
+    "@eslint-react/eslint-plugin": "^1.16.1",
     "@eslint/compat": "^1.2.2",
```

Comment:
⛏️should this be updated to 1.2.3

Reply:
@turnertime Thank you, I bumped all outdated direct and indirect dependencies to the latest versions as of 2024-11-14.